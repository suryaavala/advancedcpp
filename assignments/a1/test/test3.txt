1
1 repeat
2 add
2 repeat
3 mult
3 repeat
4 sub
4 repeat
10 sub
5 div
endrepeat
endrepeat
endrepeat
sqrt
endrepeat

1.0
1 repeat
2 add
2 repeat
3 mult
3 repeat
4 sub
4 repeat
10 sub
5 div
endrepeat
endrepeat
endrepeat
sqrt
endrepeat
