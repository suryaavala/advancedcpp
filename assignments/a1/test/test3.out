2 + 1 = 3
3 * 3 = 9
4 - 9 = -5
10 - -5 = 15
5 / 15 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
4 - 0 = 3
10 - 3 = 6
5 / 6 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
4 - 0 = 3
10 - 3 = 6
5 / 6 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
3 * 0 = 1
4 - 1 = 2
10 - 2 = 7
5 / 7 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
4 - 0 = 3
10 - 3 = 6
5 / 6 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
4 - 0 = 3
10 - 3 = 6
5 / 6 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
10 - 0 = 9
5 / 9 = 0
sqrt 0 = 0
2 + 1.000 = 3.000
3 * 3.000 = 9.000
4 - 9.000 = -5.000
10 - -5.000 = 15.000
5 / 15.000 = 0.333
10 - 0.333 = 9.667
5 / 9.667 = 0.517
10 - 0.517 = 9.483
5 / 9.483 = 0.527
10 - 0.527 = 9.473
5 / 9.473 = 0.528
4 - 0.528 = 3.472
10 - 3.472 = 6.528
5 / 6.528 = 0.766
10 - 0.766 = 9.234
5 / 9.234 = 0.541
10 - 0.541 = 9.459
5 / 9.459 = 0.529
10 - 0.529 = 9.471
5 / 9.471 = 0.528
4 - 0.528 = 3.472
10 - 3.472 = 6.528
5 / 6.528 = 0.766
10 - 0.766 = 9.234
5 / 9.234 = 0.541
10 - 0.541 = 9.459
5 / 9.459 = 0.529
10 - 0.529 = 9.471
5 / 9.471 = 0.528
3 * 0.528 = 1.584
4 - 1.584 = 2.416
10 - 2.416 = 7.584
5 / 7.584 = 0.659
10 - 0.659 = 9.341
5 / 9.341 = 0.535
10 - 0.535 = 9.465
5 / 9.465 = 0.528
10 - 0.528 = 9.472
5 / 9.472 = 0.528
4 - 0.528 = 3.472
10 - 3.472 = 6.528
5 / 6.528 = 0.766
10 - 0.766 = 9.234
5 / 9.234 = 0.541
10 - 0.541 = 9.459
5 / 9.459 = 0.529
10 - 0.529 = 9.471
5 / 9.471 = 0.528
4 - 0.528 = 3.472
10 - 3.472 = 6.528
5 / 6.528 = 0.766
10 - 0.766 = 9.234
5 / 9.234 = 0.541
10 - 0.541 = 9.459
5 / 9.459 = 0.529
10 - 0.529 = 9.471
5 / 9.471 = 0.528
sqrt 0.528 = 0.727
